RecoMuon/MuonIdentification/plugins/MuonIdProducer.cc

```diff
@@ -925,8 +925,10 @@ void MuonIdProducer::fillMuonId(edm::Event& iEvent,
                                      << ", chamber y: " << matchedChamber.y << ", max: " << maxAbsDy_;
       const double matchedSegChDx = std::abs(matchedSegment.x - matchedChamber.x);
       const double matchedSegChDy = std::abs(matchedSegment.y - matchedChamber.y);
-      const double matchedSegChPullX = matchedSegChDx / std::hypot(matchedSegment.xErr, matchedChamber.xErr);
-      const double matchedSegChPullY = matchedSegChDy / std::hypot(matchedSegment.yErr, matchedChamber.yErr);
+      const double matchedSegChPullX =
```

Comment:
if used only in comparison one could avoid division and sqrt (as done in line 1000)